### PR TITLE
ISPN-15156 Fix logging of tracing enabled

### PR DESCRIPTION
**Pull request by fax4ever**
https://issues.redhat.com/browse/ISPN-15156

### CODE REVIEW

File: client/hotrod-client/src/main/java/org/infinispan/client/hotrod/telemetry/impl/TelemetryServiceFactory.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+   }
+
+   public TelemetryService telemetryService(boolean tracingPropagationEnabled) {
+      if (tracingPropagationEnabled) {
+         return telemetryService;
+      }
+
+      if (loggedDisabledTracing == false) {
+         loggedDisabledTracing = true;
+         log.openTelemetryPropagationDisabled();
```
Comment (tristantarrant) [inline client/hotrod-client/src/main/java/org/infinispan/client/hotrod/telemetry/impl/TelemetryServiceFactory.java:26]: JBoss Logging has a  `@Once` annotation. See https://github.com/infinispan/infinispan/blob/main/server/hotrod/src/main/java/org/infinispan/server/hotrod/logging/Log.java#L56 for an example

**Review by tristantarrant — CHANGES_REQUESTED** [review]

File: client/hotrod-client/src/main/java/org/infinispan/client/hotrod/telemetry/impl/TelemetryServiceFactory.java (hunk 1 above)
Comment (fax4ever) [reply client/hotrod-client/src/main/java/org/infinispan/client/hotrod/telemetry/impl/TelemetryServiceFactory.java:26]: cool

**Comment by ryanemerson** [comment]
Thanks @fax4ever

**Comment by fax4ever** [comment]
Thank you @ryanemerson!

**Pull request merged**